pkg/clustermanager/cluster_manager.go

```diff
@@ -1167,6 +1208,37 @@ func (c *ClusterManager) CreateEKSAResources(ctx context.Context, cluster *types
 	return c.ApplyReleases(ctx, clusterSpec, cluster)
 }
 
+// CreateEKSAReleaseBundle applies the eks-a release bundle to the cluster.
+func (c *ClusterManager) CreateEKSAReleaseBundle(ctx context.Context, cluster *types.Cluster, clusterSpec *cluster.Spec) error {
```

Comment:
do we really need the wrapper?

Reply:
I ended up using this wrapper because while creating the cluster's namespace we use the cluster Client that is a private Retrier Client used by the cluster manager.